components/camel-spring-boot/src/main/java/org/apache/camel/spring/boot/CamelAutoConfiguration.java

```diff
@@ -77,7 +77,7 @@
     /**
      * Spring-aware Camel context for the application. Auto-detects and loads all routes available in the Spring context.
      */
-    @Bean
+    @Bean(destroyMethod = "")
```

Comment:
We may have done this on some other places AFAIR, maybe look if that is the case and see if we did not add some code comments why we do this, as its confusing why that string is empty